### PR TITLE
chore: fix ruff RUF012

### PR DESCRIPTION
**Pull request by chemelli74**
## Summary by CodeRabbit

- **Refactor**
  - Added `ClassVar` type hints to multiple attributes across various device classes for improved type safety.
  - Removed redundant `_fresh_air_fan_speeds_rev` attribute from `MideaACDevice` class.

- **Tests**
  - Updated assertion value for `fresh_air_mode` attribute from "Medium" to "Low" in AC device tests.

### CODE REVIEW

File: midealocal/devices/a1/__init__.py [hunk 1]
```diff
@@ -31,16 +31,22 @@ class DeviceAttributes(StrEnum):
 
 
 class MideaA1Device(MideaDevice):
-    _modes = ["Manual", "Continuous", "Auto", "Clothes-Dry", "Shoes-Dry"]
-    _speeds = {
+    _modes: ClassVar[list[str]] = [
```
Comment (rokam) [inline midealocal/devices/a1/__init__.py:34]:
```suggestion
    _modes: list[str] = [
```
Do this work?

Comment (chemelli74) [reply midealocal/devices/a1/__init__.py:34]: Nope unfortunately, this is the ref. <https://docs.astral.sh/ruff/rules/mutable-class-default/>

File: midealocal/devices/ac/__init__.py [hunk 2]
```diff
@@ -62,15 +62,17 @@ class DeviceAttributes(StrEnum):
...
         20: "Silent",
         40: "Low",
         60: "Medium",
         80: "High",
         100: "Full",
     }
-    _fresh_air_fan_speeds_rev = dict(reversed(_fresh_air_fan_speeds.items()))
+    _fresh_air_fan_speeds_rev: ClassVar[dict[str, int]] = {
+        k: v for v, k in _fresh_air_fan_speeds.items()
+    }
```
Comment (rokam) [inline midealocal/devices/ac/__init__.py]: This doesn't seem right. _fresh_air_fan_speeds_rev should be:
{
100: "Full",
80: "High",
...
0: "Off",
}

**Review by rokam — CHANGES_REQUESTED** [review]

File: midealocal/devices/ac/__init__.py (hunk 2 above)
Comment (chemelli74) [reply midealocal/devices/ac/__init__.py]: Is now removed

File: midealocal/devices/ac/__init__.py [hunk 3]
```diff
@@ -190,8 +189,8 @@ def process_message(self, msg: bytes) -> dict[str, Any]:
                 new_status[str(status)] = self._attributes[status]
         if has_fresh_air:
             if self._attributes[DeviceAttributes.fresh_air_power]:
-                for k, v in MideaACDevice._fresh_air_fan_speeds_rev.items():
-                    if self._attributes[DeviceAttributes.fresh_air_fan_speed] > k:
+                for k, v in MideaACDevice._fresh_air_fan_speeds.items():
+                    if self._attributes[DeviceAttributes.fresh_air_fan_speed] < k:
```
Comment (rokam) [inline midealocal/devices/ac/__init__.py:193]:
```suggestion
                    if self._attributes[DeviceAttributes.fresh_air_fan_speed] <= k:
```

**Review by rokam — CHANGES_REQUESTED** [review]

**Review by rokam — APPROVED** [review]

File: midealocal/devices/ac/__init__.py [hunk 4]
```diff
@@ -190,8 +189,8 @@ def process_message(self, msg: bytes) -> dict[str, Any]:
                 new_status[str(status)] = self._attributes[status]
         if has_fresh_air:
             if self._attributes[DeviceAttributes.fresh_air_power]:
-                for k, v in MideaACDevice._fresh_air_fan_speeds_rev.items():
-                    if self._attributes[DeviceAttributes.fresh_air_fan_speed] > k:
+                for k, v in MideaACDevice._fresh_air_fan_speeds.items():
+                    if self._attributes[DeviceAttributes.fresh_air_fan_speed] <= k:
```
Comment (rokam) [inline midealocal/devices/ac/__init__.py:193]:
```suggestion
                    if self._attributes[DeviceAttributes.fresh_air_fan_speed] < k:
```

**Review by rokam — APPROVED** [review]

**Pull request merged**